experimental/cmd/hd44780/main.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/gpio/gpioreg"
+	"periph.io/x/periph/experimental/devices/hd44780"
+	"periph.io/x/periph/host"
+)
+
+var pinPattern = "no %s pin specified. Please provide the pin via '%s' flag, for example '%s'"
```

Comment:
Make it a const inside mainFunc() around line 33, since it's not used elsewhere.